site/index.xml

```diff
@@ -187,10 +187,10 @@ limitations under the License.
               <h2>Commercial Distribution</h2>
               <p>
                 VMware offers a <a href="https://tanzu.vmware.com/rabbitmq">range of commercial offerings for RabbitMQ</a>.
-                This includes a distribution called <a href="https://tanzu.vmware.com/services-marketplace/messaging-and-integration/rabbitmq">Tanzu RabbitMQ</a>,
```

Comment:
I think this information needs further clarity how about: 
As well as open source RabbitMQ, VMware offers a range of commercial offerings for RabbitMQ which you can find at [Tanzu RabbitMQ](https://tanzu.vmware.com/rabbitmq). One of these offerings is [Tanzu RabbitMQ for Kubernetes](https://network.tanzu.vmware.com/products/p-rabbitmq-for-kubernetes) that deploys on Kubernetes or your container Runtime, another offering is a version that deploys in [VMware Tanzu Application Service](https://tanzu.vmware.com/tanzu). These offerings include all of the features of the open source version, with some additional management and business continuity features. Support agreements are part of the commercial licensing.